Makefile

```
@@ -259,18 +259,18 @@ core-protogen:
 	echo "Generating protobuf for Core"
 	cd workflow-schema && buf generate
 
-rla-proto:
-	RLA_DIR=rla \
-	ls "$${RLA_DIR}/proto/v1"; \
-	for file in "$${RLA_DIR}"/proto/v1/*.proto; do \
-		cp "$$file" "workflow-schema/rla/proto/v1/"; \
+flow-proto:
```

checkmake (0.3.2)

[warning] Target body for "flow-proto" exceeds allowed length of 5 lines (7).

(maxbodylength)